analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -196,7 +196,7 @@ class SpdxDocumentFile(
                 description = packageDescription,
                 homepageUrl = pkg.homepage.mapNotPresentToEmpty(),
                 binaryArtifact = RemoteArtifact.EMPTY, // TODO: Use "downloadLocation" or "externalRefs"?
-                sourceArtifact = buildSourceArtifact(pkg), // TODO: Use "sourceInfo" or "externalRefs"?
+                sourceArtifact = buildSourceArtifact(pkg),
```

Comment:
This change again should be squashed into the (separate) commit that introduces the `buildSourceArtifact()` function.